### DISCUSSION TITLE
Add timestamps to favorite tracks

### DISCUSSION THREAD

**Discussion by printmaps** [devemux86/cruiser#659] [Q&A] [answered]
If a track from the internal track management is shared, e.g. via eMail, it doesn't contains timestamps. Timestamps are essential. Please add them.

![Screenshot_20241027_092531_Cruiser](https://github.com/user-attachments/assets/a7aa9349-af1e-4d51-b34c-428eee866171)

**Comment by devemux86** [comment]
@printmaps Thanks for the suggestion.

Timestamps exist when exporting recorded GPS tracks.
In my collection we store more of the route geometries.

**Comment by devemux86** [comment]
@printmaps Timestamps in favorite tracks will be available as part of the new:
- https://github.com/devemux86/cruiser/discussions/706

**Comment by devemux86** [comment]
Timestamps in favorite tracks are available in:
- https://github.com/devemux86/cruiser/discussions/734

**Comment by devemux86** [comment, marked as answer]
- Timestamps in favorite tracks are available in https://github.com/devemux86/cruiser/discussions/757
